### PR TITLE
Fix options yaml

### PR DESCRIPTION
**Pull request by nemski**
### Context

In PR #5 I there was an issue with my `OPTIONS` not being valid YAML.

### Changes

Introduce tests to ensure the OPTIONS is valid YAML

Wrap changes in quotes to ensure it is parsable.

### CODE REVIEW

File: delayed_job/spec/monitor_delayed_jobs_spec.rb [hunk 1]
```diff
@@ -19,6 +19,10 @@ def use_transactional_fixtures
   after(:each) do
     MonitorDelayedJobs::DelayedJob.delete_all
   end
+
+  it "has options that are valid YAML" do
+    expect{YAML.load(described_class::OPTIONS)}.not_to raise_error
```
Comment (johnsyweb) [inline delayed_job/spec/monitor_delayed_jobs_spec.rb]: I dislike `not_to raise_error` as an expectation. Specs should tell the reader what *is* the expected outcome.

File: delayed_job/monitor_delayed_jobs.rb [hunk 2]
```diff
@@ -10,14 +10,13 @@ class MonitorDelayedJobs < Scout::Plugin
     default: production
   queue_name:
     name: Queue Name
```
Comment (denmat) [inline delayed_job/monitor_delayed_jobs.rb]: What about these then? You've done it for custom loader. On phone so not looking carefully.

Comment (nemski) [inline delayed_job/monitor_delayed_jobs.rb]: Not strictly necessary to quote strings with spaces, just makes it clearer. Will do them all though.

**Comment by denmat** [comment]
:+1:

File: delayed_job/spec/monitor_delayed_jobs_spec.rb [hunk 3]
```diff
@@ -19,6 +19,10 @@ def use_transactional_fixtures
   after(:each) do
     MonitorDelayedJobs::DelayedJob.delete_all
   end
+
+  it "has options that are valid YAML" do
```
Comment (johnsyweb) [inline delayed_job/spec/monitor_delayed_jobs_spec.rb]: "`it 'can load the options into a Hash'`"?

**Comment by johnsyweb** [comment]
Thank you!

**Pull request merged**